firmware/src/io/store.cpp

```
@@ -126,6 +110,11 @@ void read_setting(char* line, Settings& settings) {
   *separator = '\0';
   const char* key = line;
   const char* text = separator + 1;
+  if (std::strcmp(key, "kit") == 0) {  // a name, not a number, and the only one here
+    const size_t length = std::strlen(text);
+    if (length > 0 && length <= engine::kKitIdLength) std::strcpy(settings.kit, text);
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)